tests/ui/rfc-2091-track-caller/error-with-main.stderr

```diff
@@ -5,6 +5,12 @@ LL | #[track_caller]
    | ^^^^^^^^^^^^^^^
 LL | fn main() {
    | --------- `main` function is not allowed to be `#[track_caller]`
+   |
+help: remove the annotation
```

Comment:
I don't think this being a note is the best choice. 

Preferably it would be a suggestion. If not, then a label is better so that it renders inline.

Reply:
I made an attempt at what you meant